storefront/test/documentation/workarea/api/storefront/segmentation_documentation_test.rb

```diff
@@ -0,0 +1,104 @@
+require 'test_helper'
+require 'workarea/api/documentation_test'
+
+module Workarea
+  module Api
+    module Storefront
+      class SegmentationDocumentationTest < DocumentationTest
+        resource 'Segmentation'
+
+        def test_and_document_session_count
+          description 'Specifying session count for segmentation'
+          route storefront_api.system_content_path('home_page')
+          explanation <<~EOS
+            Workarea supports segmenting users by number of sessions, e.g.
+            First-time vs returning visitors. To support this in the storefront
+            API, the client will be responsible for managing this since the API
+            is stateless (doesn't have cookies/session).
+
+            To get session-based segments functioning, you need to pass a
+            `segment_id` parameter in each request. Workarea will use that as
```

Comment:
this seems wrong when looking at the test beneath? you're passing `sessions`?

Reply:
Whoa what was I smoking.